sdk/storage/Azure.Storage.Blobs/AzureStorageNetMigrationV12.md

```diff
@@ -570,14 +567,37 @@ BlobUriBuilder uriBuilder = new BlobUriBuilder(StorageAccountBlobUri)
 Uri sasUri = uriBuilder.ToUri();
 ```
 
+You can also create a SAS from the client object.
```

Comment:
can we restructure the guide in such a way that `GenerateSasUri(BlobSasPermissions, DateTimeOffset)` is promoted (i.e. mentioned first) and more advanced cases follow?